.github/workflows/need_reply_remove.yml

```
@@ -7,9 +7,19 @@ on:
     types:
       - created
 
+
+permissions:
+  issues: read
+  pull-requests: read
+  contents: read
```

As these are set inside `build:` below, we can stay with no permissions outside the job.

```suggestion
permissions: {}
```